### PR TITLE
Remove endorsement verification for layers where it is not yet implemented

### PR DESCRIPTION
**Pull request by jul-sh**
PR targets OC3 demo branch. 

Currently verification for the firmware and kernel layers are not yet implemented. Previous logs were misleading, as they stated that the endorsements were being verified. This removes those logs, and comments out the unused evidence.

### CODE REVIEW

**Comment by jul-sh** [comment]
Endorsements are parsed from a proto file, presumably the verification library silently defaulted to skiping layers for whom relevant endorsements where not present.

**Comment by jul-sh** [comment]
@tiziano88 @thmsbinder for clarity, this PR does not target main, it targets the OC3 demo branch.

**Review by conradgrobler — APPROVED** [review]

File: demo_oc3_2024/evidence-endorsements.textproto [hunk 1]
```diff
@@ -1,15 +1,20 @@
 oak_restricted_kernel {
```
Comment (thmsbinder) [inline demo_oc3_2024/evidence-endorsements.textproto:1]: FWIW, consider including https://github.com/project-oak/oak/pull/4898 in your branch if you have problems. (But don't do it if you are happy with the demo overall.)

**Review by thmsbinder — APPROVED** [review]

File: demo_oc3_2024/evidence-endorsements.textproto (hunk 1 above)
Comment (jul-sh) [reply demo_oc3_2024/evidence-endorsements.textproto:1]: Very cool! Not sure we have time to get new endorsements for our our binaries though

**Pull request merged**